doc/getting-started.md

```diff
@@ -172,6 +173,30 @@ Unlike PAB, CTL obscures less of the build-balance-sign-submit pipeline for tran
 
 One major caveat to using CTL in its current state is that we have no equivalent of Plutus' `awaitTxConfirmed`. We cannot guarantee that a transaction that has been accepted into a mempool has actually been added to a block. When `Contract.Transaction.submit` returns, this is **not** a guarantee that your transaction has been accepted into a block. If transaction confirmation is critical for you, you may wish to adopt a different strategy: sleeping for a pre-determined amount of time, looping until an address contains UTxOs from the recently submitted transaction, etc.... We plan to add functionality similar to `awaitTxConfirmed` in upcoming versions of CTL.
 
+### Using compiled scripts
+
+To use your own scripts, compile them to any subdirectory in the root of CTL project (where `webpack.config.js` is located) and add a relative path to `webpack.config.js` under the `resolve.alias` section.
```

Comment:
Maybe this isn't important, but some projects have a CTL project in a subdirectory of their repo. So perhaps you should refer to the root of the repository?